<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamScheduleWindow</class>
 <widget class="QMainWindow" name="ExamScheduleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>📅 برنامه امتحانات</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="title_label">
      <property name="font">
       <font>
        <family>IRANSans UI</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #2c3e50; margin: 0;</string>
      </property>
      <property name="text">
       <string>📅 برنامه امتحانات (فقط دروس انتخابی)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="info_label">
      <property name="font">
       <font>
        <family>IRANSans UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #7f8c8d; font-style: italic; text-align: center; background: linear-gradient(135deg, #ecf0f1 0%, #bdc3c7 100%); padding: 10px; border-radius: 8px; margin: 6px;</string>
      </property>
      <property name="text">
       <string>فقط دروسی که در جدول اصلی قرار داده‌اید نمایش داده می‌شوند</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="separator">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="exam_table">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>نام درس</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>کد درس</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>استاد</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>زمان کلاس</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>زمان امتحان</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>واحد</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>محل برگزاری</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="stats_label">
      <property name="font">
       <font>
        <family>IRANSans UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: linear-gradient(135deg, #d4edda 0%, #c3e6cb 100%); border: 1px solid #c3e6cb; border-radius: 8px; padding: 6px; font-weight: 500; color: #155724;</string>
      </property>
      <property name="text">
       <string>آمار برنامه در اینجا نمایش داده می‌شود</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="explanation_label">
      <property name="font">
       <font>
        <family>IRANSans UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #7f8c8d; background: #f8f9fa; padding: 8px; border-radius: 5px; border: 1px solid #e9ecef;</string>
      </property>
      <property name="text">
       <string>توضیحات:
• زوج: دروس هفته‌های زوج (در جدول با علامت ز نشان داده شده)
• فرد: دروس هفته‌های فرد (در جدول با علامت ف نشان داده شده)
• همه هفته‌ها: دروسی که هر هفته تشکیل می‌شوند</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="bottom_separator">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_export"/>
  </widget>
  <action name="action_export">
   <property name="text">
    <string>📤 صدور برنامه امتحانات</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>